.eslintrc.json

```diff
@@ -1,4 +1,8 @@
 {
+    "globals": {
+        "fetch": true,
+        "Headers": true
+    },
```

Comment:
These global's aren't necessary. Right now, the `client` folder isn't linted. When it is, we can specify `browser: true` in the `env` object which will enable these globals automatically.

Recommended action, remove the `globals` object.